src/HttpClient/ClientOptions.php

```diff
@@ -0,0 +1,111 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\Externals\HttpClient;
+
+use EoneoPay\Externals\HttpClient\Interfaces\ClientOptionsInterface;
+use GuzzleHttp\RequestOptions;
+
+class ClientOptions implements ClientOptionsInterface
+{
+    /**
+     * @var float The number of seconds the HTTP client will wait to connect.
+     */
+    private $connectTimeout = 5;
+
+    /**
+     * @var float The number of seconds the HTTP client will wait for the response body to
+     *          stream before timing out.
+     */
+    private $readTimeout = 0;
```

Comment:
10 seconds is appropriate here - if the response doesnt start streaming within 10 seconds we're going to error out